cmd/main.go

```diff
@@ -109,7 +110,6 @@ func (c Configuration) consumers(db *pkg.AgentDB) (consumers pkg.BaseConsumers)
 		}
 		consumers = append(consumers, &pkg.BaseConsumer{AgentDB: db, ParserLoader: state})
```

Comment:
Consumers for ignored files are still created here, right?
I think we need move initial exclusion logic from `Watcher::Start` function to somewhere before that line in order to prevent creation of not-relevant consumers

Reply:
Updated and verified that the consumer list does not hold excluded files/dirs.